### ISSUE TITLE
R CMD check failures on Ubuntu 16.04 and Windows

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/reproducible#152]
failures showing up on GitHub action checks (Ubuntu 16.04):

```r
  ══ testthat results  ═══════════════════════════════════════════════════════════
  [ OK: 524 | SKIPPED: 13 | WARNINGS: 1 | FAILED: 8 ]
  1. Error: preProcess works when provides url and destinationPath for a .rar file (@test-preProcessWorks.R#134) 
  2. Error: preProcess works when provides url, targetfile and destinationPath for a .rar file (@test-preProcessWorks.R#154) 
  3. Error: preProcess works when provides url, archive and destinationPath for a .rar file (@test-preProcessWorks.R#175) 
  4. Failure: prepInputs doesn't work (part 1) (@test-prepInputs.R#100) 
  5. Error: prepInputs in a two files double nested rar file, with the wanted file in the second layer, not specifying the targetFile (@test-prepInputsInNestedArchives.R#105) 
  6. Error: prepInputs in a two files double nested rar file, with the wanted file in the second layer, specifying the targetFile (@test-prepInputsInNestedArchives.R#130) 
  7. Error: prepInputs in a two files double nested rar file, with the wanted file inthe second layer, not specifying the targetFile, passing the main archive (@test-prepInputsInNestedArchives.R#154) 
  8. Error: prepInputs works with nested rar file inside internal rar folder (@test-prepInputsInNestedArchives.R#180) 
```

- [x] error 1 above
- [x] error 2 above
- [x] error 3 above
- [x] error 4 above
- [x] error 5 above
- [x] error 6 above
- [x] error 7 above
- [x] error 8 above

**Comment by achubaty** [comment]
Additonally:

```r
Error in system2("du", tmpCache, stdout = TRUE) : '"du"' not found
```

```r
# man/cropInputs.Rd
st_buffer does not correctly buffer longitude/latitude data
# man/cropInputs.Rd
st_buffer does not correctly buffer longitude/latitude data
# man/determineFilename.Rd
st_buffer does not correctly buffer longitude/latitude data
```

```r
# man/fastMask.Rd
unable to find an interited method for function 'rasterize' for signature '"Spatial", "RasterStack"'
```

```r
test-examples.R:35: error: all exported functions have examples
cannot  open the connection
```

```r
test-gis.R:24: error: fastMask produces correct results
error in evaluating the argument 'x' in selecting a method for function 'stark': unable to find an inherited method for function 'rasterize' for signature '"Spatial", "RasterStack"'
```

```r
test-misc.R:160: failure: repo stuff works
grepl("https://cloud.r-project.org", a) isn't true

???: failure: repo stuff works
grepl("https://cloud.r-project.org", unname(out)) isn't true
```

@ianmseddy I believe you started dealing with somo of these? please push to `152-check-fixes` branch.

**Comment by achubaty** [comment]
```r
17. test-prepInputs.R:1544: error: lightweight tests 2 for code coverage
```
[ ] error 17 above

**Comment by achubaty** [comment]
merged `152-check-fixes` into `development` (1b53941ea34c129b2deca09a6eb44d1f94c033c0)
